Eth-Global-Guide.md

```diff
@@ -0,0 +1,32 @@
+# Intro to Zetachain
+
+## Introduction
+
+- What is ZetaChain?
+- Why build dApps on Zeta ? (Key features) 
+- What is ZETA token ?
+
+## Build with Zeta
+- Start building dapps ( Refer to the first half -  https://www.zetachain.com/docs/developers/overview/) 
+- Using Zetachain Smart Contracts (conceptual explanation)
+> - Omnichain Contracts
+> - Cross Chain Messaging (explanation 
+- Sending and Depositing Tokens
+- Smart Contract Template
+
+## Smart Contracts
```

Comment:
CCM and omnichain are quite different in terms of stack, so I'd do:

* Introduction
* Build with ZetaChain (things common to both CCM and omnichain)
  * Start building dapps
  * Sending and depositing tokens
  * Smrt contract template
* Omnichain contracts
* Cross-chain messaging
* Developer resources

Reply:
btw, the introduction section was fine:

## Introduction

- What is ZetaChain?
- Why build dApps on Zeta ? (Key features) 
- What is ZETA token ?